common/resource/resourceTest.go

```diff
@@ -313,8 +311,8 @@ func (s *Test) GetWorkerServiceResolver() membership.ServiceResolver {
 // internal services clients
 
 // GetSDKClient for testing
-func (s *Test) GetSDKClient() workflowservice.WorkflowServiceClient {
-	return s.SDKClient
+func (s *Test) GetSDKClient() sdkclient.Client {
+	panic("user should implement this method for test")
```

Comment:
It should not be hard to generate mock for `sdkclient.Client` but it seems not to be used anywhere.

Reply:
I am laughing at myself 4 hours ago :-)